compiler/core/src/zserio/ast/TemplateSymbol.java

```diff
@@ -1,40 +1,57 @@
 package zserio.ast;
 
-import java.math.BigInteger;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Set;
-
 import antlr.collections.AST;
 import zserio.antlr.ZserioParserTokenTypes;
 import zserio.antlr.util.BaseTokenAST;
 import zserio.antlr.util.ParserException;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * AST node for template parameters.
  */
-public class TemplateSymbol extends TokenAST
+public class TemplateSymbol extends CompoundType
 {
-    /**
-     * Gets the name of the field.
-     *
-     * @return Returns name of the field.
-     */
+    @Override
+    public Package getPackage()
```

Comment:
can't you use the actual package here (copied during parsing, like for other types)?